File: video-streaming-backend/video-service/src/main/java/com/fourbit/sachintha/dto/CommentDto.java
package com.fourbit.sachintha.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CommentDto {
	private Long id;
	private String text;
	private UserDto user;
	private Long likesCount = Long.valueOf(0);
	private Long dislikesCount = Long.valueOf(0);
	private String lifeTime;
	private UserLikeStatus userLikeStatus;
}

File: video-streaming-frontend/src/app/interfaces/comment.dto.ts
import { UserDto } from './user.dto';
import { VideoDto } from './video.dto';
import { UserLikeStatus } from './videoLikeStatus.dto';

export interface CommentDto {
	id: number;
	text: string;
	user: UserDto;
	likesCount: number;
	dislikesCount: number;
	lifeTime: string;
	userLikeStatus?: UserLikeStatus;
}
